nginx.agent.conf

```diff
@@ -0,0 +1,13 @@
+include common/main.conf;
+
+
+http {
+    include common/http.conf;
+
+    server {
+        include common/server.conf;
+
+        listen 32080 default_server;
+        listen 32443 ssl spdy default_server;
```

Comment:
I went with these ports because they're somewhat memorable and unreserved on all agents. https://github.com/dcos/dcos/blob/dedd2b4a2cc2aaaf07a78a884a93e7bba69f9cb7/gen/dcos-config.yaml#L127